### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by dobooth**
## Scope of this PR
- #4223 
- #4222 
- #4114 
- #4123 
- #4160 
- #4179 
- #4189
- #4212 
- #4218 
- #4219

### CODE REVIEW

File: guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ For each CSS file included in the layouts, Less preprocessor does the following:
 
 In server-side Less compilation mode, to have your changes applied, you need to do the following:
 
-1. Clear `pub/static/frontend/<Vendor>/<theme>/<locale>` by deleting the directory in the file system.
+1. Clear `pub/static/frontend/<Vendor>/<theme>/<locale>` by deleting the directory in the file system. (excluding .htaccess)
```
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md]:
```suggestion
1. Clear `pub/static/frontend/<Vendor>/<theme>/<locale>` by deleting the directory in the file system (excluding .htaccess).
```

File: guides/v2.2/payments-integrations/payment-gateway/gateway-command.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ In the following example the `BraintreeAuthorizeCommand` gateway command is adde
 </virtualType>
 {% endhighlight %}
 
-(The code sample is from {{site.data.var.ce}} v2.1. Although the payment provider gateway was added in v2.0, the particular default implementation using the gateway were added in v2.1)
+(The code sample is from {{site.data.var.ce}} v2.2. Although the payment provider gateway was added in v2.0, the particular default implementation using the gateway were added in v2.1)
```
Comment (shrielenee) [inline guides/v2.2/payments-integrations/payment-gateway/gateway-command.md]:
```suggestion
(The code sample is from {{site.data.var.ce}} v2.2. Although the payment provider gateway was added in v2.0, the particular default implementation using the gateway were added in v2.1.)
```

File: guides/v2.2/ui_comp_guide/concepts/ui_comp_config_flow_concept.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ Now it is the client's turn to process this JSON and generate the UI component's
 
 1. RequireJS requires `Magento_Ui/js/core/app` and passes [JSON configuration]({{ page.baseurl }}/javascript-dev-guide/javascript/js_init.html#decl_tag) as a parameter.
 2. The `Magento_Ui/js/core/app` calls `layout.js `and passes the UI component’s configuration into the layout: `<Magento_Ui_module_dir>/view/base/web/js/core/renderer/layout.js`.
-3. `layout.js` creates instances of UI components. That means that each UI component’s configuration must have an explicitly declared `component` property in JSON. This property references the `.js` file. For example, our form has the component declared in JSON like this:	`"my_form":{"component":"Magento_Ui/js/form/form"}`
+3. `layout.js` creates instances of UI components. That means that each UI component’s configuration must have an explicitly declared `component` property in JSON. This property references the `.js` file. For example, our form has the component declared in JSON like this: `"my_form":{"component":"Magento_Ui/js/form/form"}`
```
Comment (shrielenee) [inline guides/v2.2/ui_comp_guide/concepts/ui_comp_config_flow_concept.md]:
```suggestion
3. `layout.js` creates instances of UI components. That means that each UI component’s configuration must have an explicitly declared the `component` property in JSON. This property references the `.js` file. For example, our form has the component declared in JSON like this: `"my_form":{"component":"Magento_Ui/js/form/form"}`
```

**Comment by dobooth** [comment]
running tests

**Pull request merged**